### PR TITLE
[env/ohbm] add venue logos to poster hall template

### PR DESCRIPTION
**Pull request by denisdimitrov**
Conditionally renders logos in the poster hall. Left, right, both or none.

Fixes: https://github.com/sparkletown/internal-sparkle-issues/issues/694

posterhall

### CODE REVIEW

File: src/settings.ts [hunk 1]
```diff
@@ -149,6 +149,8 @@ export const PLAYA_VENUE_STYLES: Record<string, CSSProperties> = {
   },
 };
 
+export const POSTER_LOGO_SIZE = 100;
```
Comment (mike-lvov) [inline src/settings.ts:152]: Any reason to have it like this rather than in classes

**Review by mike-lvov — CHANGES_REQUESTED** [review]
And I think it should be based off `env/ohbm` as the changes are very specific

File: src/settings.ts (hunk 1 above)
Comment (denisdimitrov) [reply src/settings.ts:152]: Not really, I'm just using it with img's width prop like this ``

**Comment by denisdimitrov** [comment]
> And I think it should be based off env/ohbm as the changes are very specific

This PR is going in `staging` instead of `env/ohbm` maybe I should change it?

**Comment by mike-lvov** [comment]
> This PR is going in staging instead of env/ohbm maybe I should change it?

Yes

**Comment by denisdimitrov** [comment]
Merged with `env/ohbm`

File: src/settings.ts (hunk 1 above)
Comment (mike-lvov) [reply src/settings.ts:156]: Can we do it with the classes just to follow the patterns?

**Review by mike-lvov — APPROVED** [review]
LGTM

**Comment by 0xdevalias** [comment]
> > This PR is going in staging instead of env/ohbm maybe I should change it?
> 
> Yes

@mike-lvov @denisdimitrov FWIW (and we can assess this in the future when we have more capacity), this doesn't seem like a bad idea to have in the main code branch for the poster hall.

**Pull request merged**